jib-maven-plugin/build.gradle

```diff
@@ -12,32 +12,32 @@ dependencies {
   sourceProject project(':jib-plugins-common')
   ensureNoProjectDependencies()
 
-  implementation "com.google.cloud.tools:jib-maven-plugin-extension-api:${dependencyVersions.MAVEN_EXTENSION}"
+  implementation dependencyStrings.MAVEN_EXTENSION
 
-  implementation "org.apache.maven:maven-plugin-api:${dependencyVersions.MAVEN_API}"
-  implementation "org.apache.maven:maven-core:${dependencyVersions.MAVEN_API}"
+  implementation dependencyStrings.MAVEN_API
+  implementation dependencyStrings.MAVEN_CORE
 
   // compileOnly + testImplementation equivalent to "provided"
-  compileOnly 'org.apache.maven.plugin-tools:maven-plugin-annotations:3.6.0'
+  compileOnly dependencyStrings.MAVEN_ANNOTATIONS_LATEST
   // needed to suppress "unknown enum constant" warnings
-  testImplementation 'org.apache.maven.plugin-tools:maven-plugin-annotations:3.5'
+  testImplementation dependencyStrings.MAVEN_ANNOTATIONS_OLD
 
   // maven-plugin-testing-harness pulls in conflicting implementations of DefaultPlexusContainer
   // (sisu (correct) vs default-plexus-container (wrong)) so ensure this is first in the test classpath
-  testImplementation 'org.eclipse.sisu:org.eclipse.sisu.plexus:0.3.4'
-  testImplementation 'org.apache.maven.plugin-testing:maven-plugin-testing-harness:3.3.0'
+  testImplementation dependencyStrings.SISU_PLEXUS
+  testImplementation dependencyStrings.MAVEN_TESTING_HARNESS
 
-  testImplementation "junit:junit:${dependencyVersions.JUNIT}"
-  testImplementation "org.mockito:mockito-core:${dependencyVersions.MOCKITO_CORE}"
-  testImplementation "org.slf4j:slf4j-api:${dependencyVersions.SLF4J_API}"
-  testImplementation "com.github.stefanbirkner:system-rules:${dependencyVersions.SYSTEM_RULES}"
+  testImplementation dependencyStrings.JUNIT
+  testImplementation dependencyStrings.MOCKITO_CORE
+  testImplementation dependencyStrings.SLF4J_API_OLD
+  testImplementation dependencyStrings.SYSTEM_RULES
 
-  testImplementation 'org.apache.maven.shared:maven-verifier:1.7.2'
-  testImplementation "org.apache.maven:maven-compat:${dependencyVersions.MAVEN_API}"
-  testImplementation 'org.slf4j:slf4j-api:1.7.30'
-  testImplementation 'org.slf4j:slf4j-simple:1.7.30'
+  testImplementation dependencyStrings.MAVEN_VERIFIER
+  testImplementation dependencyStrings.MAVEN_COMPAT
+  testImplementation dependencyStrings.SLF4J_API_LATEST
```

Comment:
hrmm weird that these are both in here, can you file a new issue to handle standardizing the SLF4J apis?

Reply:
Sure! Filed issue #2608